data/camvid.lua

```diff
@@ -22,8 +20,8 @@ function CamvidDataset:get(i)
     local image_path = ffi.string(self.imageInfo.imagePath[i]:data())
     local label_path = ffi.string(self.imageInfo.labelPath[i]:data())
 
-    local image = self:_loadImage(paths.concat(self.dir, image_path), 3)
-    local label = self:_loadImage(paths.concat(self.dir, label_path), 1):squeeze():float() + 2
+    local image = self:_loadImage(image_path, 3)
```

Comment:
I am not sure of that. The expectation is to use self.dir as the root folder and image_path to keep the relative folder for the corresponding image class. Therefore, I believe it is better off to use with the previous convention.